doc/Caliptra.md

```diff
@@ -374,14 +374,14 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
 2. HVM programs NIST compliant UDS into fuses using SoC-specific fuse programming flow. Note that this UDS goes through an obfuscation function within Caliptra IP.
 3. SoC drives the security state, which indicates that it's a manufacturing flow. See [Caliptra Security States](#caliptra-security-states) for encodings.
 4. SoC (using a GPIO pin or SoC ROM) drives BootFSMBrk (this is also used for debug cases). This can be driven at any time before cptra\_rst\_b is deasserted.
-5. SoC follows the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers.
+5. SoC follows the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers. SoC shall not attempt to initiate any mailbox operation or acquire mailbox lock via the APB interface until after the subsequent CSR flow is completed.
 6. HVM, through JTAG or using the Caliptra SoC interface, sets “CPTRA\_DBG\_MANUF\_SERVICE\_REG” bit 0 to request a CSR.
 7. HVM, through JTAG or using the Caliptra SoC interface, writes to “CPTRA\_BOOTFSM\_GO” to allow Caliptra’s internal BootFSM to continue to bring up microcontroller out of reset.
-8. ROM looks at the manufacturing state encoding, “CPTRA\_DBG\_MANUF\_SERVICE\_REG”, and populates the Caliptra internal SRAM \[the mailbox SRAM hardware structure is reused\] with the CSR.
+8. ROM looks at the manufacturing state encoding, “CPTRA\_DBG\_MANUF\_SERVICE\_REG”, acquires the mailbox lock, and populates the Caliptra internal SRAM \[the mailbox SRAM hardware structure is reused\] with the CSR.
```

Comment:
Feel free to ignore if out of scope for this PR. If not, then I recommend updating the following: 

-  `looks at the manufacturing state encoding, "CPTRA\_DBG\_MANUF\_SERVICE\_REG"` to `verifies that the manufacturing state encoding is "CPTRA\_DBG\_MANUF\_SERVICE\_REG"`
- change square brackets to parentheses